rest/model/test.go

```diff
@@ -89,6 +89,10 @@ func (at *APITest) BuildFromService(st interface{}) error {
 			dispString := fmt.Sprintf("/test_log/%s?text=true", *at.Logs.LogId)
 			at.Logs.RawDisplayURL = &dispString
 		}
+
+	    if v.GroupID != "" {
```

Comment:
what is the best way to format a Go file?

Reply:
Theres a built in go linter. My editor runs it on save. maybe try `make lint` https://github.com/evergreen-ci/evergreen/blob/e533ec058611329080f2df560da2dd55871c042e/makefile#L239

Reply:
you can run make lint-rest-model to just run it for this package, but it doesn't format it, it's just running lint locally. I would fine a format plugin or something for your editor.

Reply:
Use [Gofmt](https://blog.golang.org/gofmt), there should be a plugin for your editor.

Reply:
sweet Gofmt and lint work pretty well!!